manager/integration/tests/test_upgrade.py

```diff
@@ -70,6 +75,17 @@ def upgrade_longhorn_backing_image_manager_image(request):
     return request.config.getoption("--upgrade-lh-backing-image-manager-image")
 
 
+def get_longhorn_upgrade_type():
+    return [os.environ.get('LONGHORN_UPGRADE_TYPE', '')]
+
+
+@pytest.fixture(params=get_longhorn_upgrade_type())
+def longhorn_upgrade_type():
+    # add parameter "from_stable" or "from_transient" to test_upgrade test case
+    # to distinguish them in the junit report.
+    pass
```

Comment:
Do you know what the purpose here? Why this can help with distinguish the report? (I approved before, but it seems not getting clarified now)

Reply:
Hi @innobead:
     After I checked the [PR-1179](https://github.com/longhorn/longhorn-tests/pull/1179), we support it since 1.4.x.
     I think I should remove this change from the1.3.x.